### DISCUSSION TITLE
Améliorer l'esthétique du top

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#1937] [Nouvelles idées]
L'idée serait de rajouter dans le profil des joueur leurs classe en emoji

Voici quelque exemple que j'ai essayé
(juste pour avoir une idée de ce à quoi cela pourrait ressembler)

![Screenshot_20230425_160914](https://user-images.githubusercontent.com/96150225/234304361-ddb05925-e488-45d5-afdf-3c5d9f9ae909.jpg)
![Screenshot_20230425_160924](https://user-images.githubusercontent.com/96150225/234304481-6bf28489-86a0-4377-bef5-1ec043d24314.jpg)
![Screenshot_20230425_160936](https://user-images.githubusercontent.com/96150225/234304549-3d38e722-2c45-433c-be84-5375cc4b684a.jpg)
![Screenshot_20230425_160946](https://user-images.githubusercontent.com/96150225/234304602-55f685a5-d939-45ea-9262-27008a14a505.jpg)

J'ai abrégé le Niveau en Niv pour éviter que ça prenne trop de place
Peut être à faire avec le Level et lvl si c'est pris en compte

**Comment by symsym-1629** [comment]
why not mais ca reste très mineur, ca risque pas d'être dans les priorités

**Reply by Bombafox** [reply]
Oui je me doute avec tout ce qu'il y'a en ce moment
Mais je voulais juste le mettre par la
